src/utils/operator.ts

```diff
@@ -44,15 +44,35 @@ function isOrderPresigning(order: RawOrder): boolean {
   return order.status === 'presignaturePending'
 }
 
+/**
+ * An order is considered cancelled if the `invalidated` flag is `true` and
+ * it has been at least `PENDING_ORDERS_BUFFER` since it has been created.
+ * The buffer is used to take into account race conditions where a solver might
+ * execute a transaction after the backend changed the order status.
+ *
+ * We assume the order is not fulfilled.
+ */
+function isOrderCancelled(order: Pick<RawOrder, 'creationDate' | 'invalidated'>): boolean {
+  const PENDING_ORDERS_BUFFER = 60 * 1000 //60s
```

Comment:
Shouldn't this be a global constant?

Reply:
I have moved it to a global constant to make it available.